### PR TITLE
STUD-804: Add Workiva CI

### PR DESCRIPTION
**Pull request by andrewunderwood-wf**
## Ticket

https://jira.atl.workiva.net/browse/STUD-804

Sets up Basic Workiva CI processes
- Dockerfile
- skynet.yaml
- dependabot
- aviary

## QA
- [] Build artifacts contain: 
    - [] anaplan-connect-[VERSION]-SNAPSHOT-javadoc.jar
    - [] anaplan-connect-[VERSION]-SNAPSHOT.jar (jar to satisfy RM's audit requirements)
- [] Package `anaplan-connect` appears on `maven-dev-internal` in Artifactory 
- [] Skynet results show unit tests are running and passing
- [] Sample regression chain passes with the new artifact
     - TODO - locate regression chain

### CODE REVIEW

**Comment by aviary-wf** [comment]
## Security Insights
The items listed below may not capture all security relevant changes. Before providing a security review, be sure to review the entire PR for security impact.
<details>
<summary>(6) Vulnerable direct dependencies were detected</summary>
<li>1 vulns in <code>com.fasterxml.jackson.core:jackson-databind < 2.12.6.1</code> via <code>java/pom.xml</code></li>
<li>1 vulns in <code>com.google.guava:guava</code> via <code>java/pom.xml</code> with no fix reported by GitHub</li>
<li>1 vulns in <code>commons-io:commons-io < 2.7</code> via <code>java/pom.xml</code></li>
<li>1 vulns in <code>junit:junit < 4.13.1</code> via <code>java/pom.xml</code></li>
<li>2 vulns in <code>mysql:mysql-connector-java < 8.0.28</code> via <code>java/pom.xml</code></li>
<li>3 vulns in <code>com.h2database:h2 < 2.1.210</code> via <code>java/pom.xml</code></li>
</details>
<details>
<summary>(2) Security relevant changes were detected</summary>
<li>Watched file <code>aviary.yaml</code> added</li>
<li>Watched keyword <code>BUILD_ARTIFACTS_VERACODE</code> in <code>Dockerfile</code> line(s) ['25'] added</li>
</details>

## Action Items
* Request infosec review via Slack: #support-infosec
* Review dependencies for available updates
* See <a href="https://splunk.workiva.net/en-US/app/search/security_vulnerable_dependencies?form.repository=anaplan-java-client">this Splunk dashboard</a> for more CVE details
* Verify [`aviary.yaml`](https://wiki.atl.workiva.net/pages/viewpage.action?pageId=42762752#SecurityInsightFramework&Tooling-ConfiguringAviary) coverage of security relevant code
---
_Questions or Comments? Reach out on Slack: #support-infosec._

File: skynet.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+description: run anaplan-java-client unit tests
+timeout: 300 #5 minutes
+size: small
+contact: support-onecloud
+
+run:
+  on-pull-request: yes
+  on-tag: yes
+  when-branch-name-is:
+    - ^refs/tags/.+
```
Comment (chrisgustavsen-wf) [inline skynet.yaml:13]: If `on-tag` is true, then do you need `when-branch-name-is: - ^refs/tags/.+`

**Review by joecotton-wk — APPROVED** [review]
+10

* Skynet is running tests correctly
* Build is working correctly

**Comment by joecotton-wk** [comment]
grg

**Comment by joewaters-wf** [comment]
@joecotton-wk and I had some conversations about this on Friday, I'm not sure this is publishing the artifacts in the way we are expecting or need to for proper consumption. Will continue conversations today.

**Review by joewaters-wf — APPROVED** [review]

**Comment by joewaters-wf** [comment]
QA +1
- Skynet is running the test config and reporting tests properly
- Build artifact exists on Artifactory as a package
- Not consumed directly by chains, will be tested as part of the migration of `plugin-anaplan-automation` (STUD-791)

**Comment by joecotton-wk** [comment]
@Workiva/release-management-p

**Review by rmconsole-wf — APPROVED** [review]
+1 from RM

**Pull request merged**